platform/security/interceptor/security-interceptor-guest/src/main/java/org/codice/ddf/security/interceptor/GuestInterceptor.java

```diff
@@ -92,11 +91,12 @@ public void handleMessage(SoapMessage message) throws Fault {
 
   private synchronized Subject getSubject(String ipAddress) {
     Subject subject = cachedGuestSubjectMap.get(ipAddress);
-    if (Security.getInstance().tokenAboutToExpire(subject)) {
+    if (subject == null) {
       GuestAuthenticationToken token = new GuestAuthenticationToken(ipAddress);
       LOGGER.debug("Getting new Guest user token for {}", ipAddress);
       try {
         subject = securityManager.getSubject(token);
+        // this should be a cache not a map so we can remove items, make this change
```

Comment:
Do we need to make a ticket for this or track it someway?

Reply:
Ah ha! See, I put that in there for myself. Thanks. I need to change that.